### PR TITLE
Use BBC Reith Qalam font sizes and line-heights

### PR DESCRIPTION
**Pull request by simonsinclair**
No issue.

**Overall change:** 

This PR changes the default script font sizes and line-heights to use those tuned for BBC Reith Qalam.

**Code changes:**

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [ ] ~~This PR requires manual testing~~

### CODE REVIEW

File: packages/utilities/gel-foundations/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 6.0.0 | [PR#????](https://github.com/bbc/psammead/pull/????) Use BBC Reith Qalam font sizes and line-heights, and remove `arabicPashto` script in favour or `arabic`. |
```
Comment (simonsinclair) [inline packages/utilities/gel-foundations/CHANGELOG.md]:
```suggestion
| 6.0.0 | [PR#4008](https://github.com/bbc/psammead/pull/4008) Use BBC Reith Qalam font sizes and line-heights, and remove `arabicPashto` script in favour or `arabic`. |
```

File: packages/utilities/gel-foundations/README.md [hunk 2]
```diff
@@ -56,8 +56,7 @@ In addition to Latin script defaults, Psammead supports typography for a number
 | --------------------------- | -------- | --------------------------- |
 | latin-and-cyrillic          | News, Hausa, Gahuza, Somali, Swahili, Igbo, Pidgin, Afaan Oromo, Yoruba, Azeri, Indonesia, Uzbek, Serbian, Russian, Ukrainian, Kyrgyz, Uzbek | The typography values for Latin and Cyrillic characters are identical |
 | latin-with-diacritics | Vietnamese, Mundo, Afrique, Brasil, Turkish | The diacritics make these values differ from Latin |
-| arabic                      | Urdu, Arabic, Persian | Group as they use the Arabic alphabet |
-| arabic-pashto               | Pashto | Contains a subset of glyphs that mean it differs from Arabic |
+| arabic                      | Arabic, Pashto, Persian, Urdu, | Group as they use the Arabic alphabet |
```
Comment (karinathomasbbc) [inline packages/utilities/gel-foundations/README.md]:
```suggestion
| arabic                      | Arabic, Pashto, Persian, Urdu | Group as they use the Arabic alphabet |
```

**Review by karinathomasbbc — APPROVED** [review]

File: packages/utilities/gel-foundations/src/scripts.js [hunk 3]
```diff
@@ -1,28 +1,47 @@
...
+  latin,
+  latinDiacritics,
+  chinese,
+  ethiopic,
+  korean,
+  noAscendersOrDescenders,
+  sinhalese,
+  tamil,
+  thai,
+};
```
Comment (andrewscfc) [inline packages/utilities/gel-foundations/src/scripts.js:47]: What is the reason for the change in format; to import then export in a seperate object?

File: packages/utilities/gel-foundations/src/scripts/arabic.js [hunk 4]
```diff
@@ -71,142 +71,142 @@ const arabicTypography = {
   },
   canon: {
     groupA: {
-      fontSize: '32',
-      lineHeight: '42',
+      fontSize: '24',
```
Comment (andrewscfc) [inline packages/utilities/gel-foundations/src/scripts/arabic.js:74]: Is there a doc you could link me with these values that I can check them against? I think I looked at a similar PR for @karinathomasbbc the last time we tried this and there was a spreadsheet?

File: packages/utilities/gel-foundations/tests/getTypeSizes.test.js [hunk 5]
```diff
@@ -1,25 +1,30 @@
...
-    }
-
-    ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
-      font-size: 2.75rem;
-      line-height: 3.375rem;
-    }
-  `;
-
-    expect(block).toEqual(canonStyles);
+      expect(styles).toMatchSnapshot();
```
Comment (andrewscfc) [inline packages/utilities/gel-foundations/tests/getTypeSizes.test.js:27]: I'm not really sure what these snapshot tests add here? They appear to show that font size and line height successfully translate into suitable `rem` values, could this be done with one test for `getTypeSizes` rather than one snapshot for each `typography` value?

File: packages/utilities/gel-foundations/src/scripts/arabic.js (hunk 4 above)
Comment (karinathomasbbc) [reply packages/utilities/gel-foundations/src/scripts/arabic.js:74]: These are the same changes as I made previously: https://docs.google.com/spreadsheets/d/1YIUyAS_WyrPudH-RTdVesFtUKvhNGhVejRSRThnCByw/edit

File: packages/utilities/gel-foundations/src/scripts.js (hunk 3 above)
Comment (karinathomasbbc) [reply packages/utilities/gel-foundations/src/scripts.js:47]: Lots of linting errors / red sqiggly lines in VSCode eek. I can put this back if you like, although it is a pattern that we use quite often in Simorgh - e.g. https://github.com/bbc/simorgh/blob/latest/src/integration/common/index.js

Comment (simonsinclair) [reply packages/utilities/gel-foundations/src/scripts.js:47]: This is the current standard way of 're-exporting'. The other method requires Babel transpilation. Whilst we have the Babel plugin installed that allows for this, it is subject to change. This PR favours the current standard.

File: packages/utilities/gel-foundations/tests/getTypeSizes.test.js (hunk 5 above)
Comment (simonsinclair) [reply packages/utilities/gel-foundations/tests/getTypeSizes.test.js:27]: The idea behind the snapshot tests here are to also 'freeze' the current font-size and line-height values of all type sizes. We weren't tracking changes to these and felt it was a good idea to improve coverage here.

Comment (andrewscfc) [reply packages/utilities/gel-foundations/tests/getTypeSizes.test.js:27]: Tbh, I kinda feel we can track changes like this through code reviews as it would need a review to change the values anyway. Happy to approved though :)

**Review by andrewscfc — APPROVED** [review]
